app/src/main/java/org/mozilla/fenix/onboarding/FenixOnboarding.kt

```diff
@@ -23,6 +23,7 @@ class FenixOnboarding(context: Context) {
 
     fun finish() {
         onboardingPrefs.onboardedVersion = CURRENT_ONBOARDING_VERSION
+        // TODO: This currently seems to be sent every time I test on debug...
```

Comment:
Any ideas what's going on here 🤔

Reply:
Hmm, looks like this get's called even if onboarding isn't visible. We should probably fix in a follow up issue :)

Reply:
Filed follow up #4824